src/screens/bookings/Complaint.tsx

```diff
@@ -1,18 +1,57 @@
-import React from "react";
-import { View, Button, Text } from "react-native";
+import React, { useState } from "react";
+import { View, Text, TextInput, ScrollView, TouchableOpacity } from "react-native";
 import { useNavigation } from '@react-navigation/native';
 import { NativeStackNavigationProp } from "@react-navigation/native-stack";
 
 export function ComplaintScreen() {
     const navigation = useNavigation<NativeStackNavigationProp<any>>();
+    const [complaintTopic, setComplaintTopic] = useState("");
+    const [complaintText, setComplaintText] = useState("");
+    const [messageSent, setMessageSent] = useState(false);
+
+    const submitComplaint = () => {
+        console.log("Topic:", complaintTopic);
+        console.log("Complaint:", complaintText);
+        setMessageSent(true);
+
+    
+        setComplaintTopic("");
+        setComplaintText("");
+    };
 
     return (
-        <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
-            <Text>Complaint</Text>
-            <Button
-                title="Go to back"
-                onPress={() => navigation.goBack()}
-            />
-        </View>
+        <ScrollView style={{ flex: 1 }}>
+            <View style={{ padding: 20 }}>
+                {messageSent && (
+                    <Text style={{ backgroundColor: "lightgreen", padding: 10, borderRadius: 5, fontSize: 16, color: "green", textAlign: 'center', marginVertical: 10 }}>
```

Comment:
Please use Tailwind instead of styling
```suggestion
                    <Text className="bg-lightgreen-500 p-10 etc...">
```